app/controllers/lead_providers/your_schools_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
                            @cohorts.find_by(start_year: Time.zone.today.year)
                          end
 
-      @schools = School.partnered_with_lead_provider(@lead_provider.id, @selected_cohort.start_year)
+      @schools = School.eligible.partnered_with_lead_provider(@lead_provider.id, @selected_cohort.start_year)
```

Comment:
Yes, in my testing I had unwittingly added a school that wasn't eligible to a partnership and it was showing in my list of schools. Whether that would happen in real-life or whether an eligible school could close/merge/split/academise or otherwise change so it is no longer eligible is a good question.